pdc_client/__init__.py

```diff
@@ -91,27 +91,71 @@ def read_config_file(server_alias):
 
 
 class PDCClient(object):
-    """BeanBag wrapper specialized for PDC access.
+    """
+    BeanBag wrapper specialized for PDC access.
 
     This class wraps general BeanBag.v1 objects, but provides easy-to-use
     interface that can use configuration files for specifying server
     connections. The authentication token is automatically retrieved (if
     needed).
     """
+
+    # Set a page_size which is used in _WrapperBeanBag
+    page_size = None
```

Comment:
I'm assuming this should be `DEFAULT_PAGE_SIZE`? This is probably not needed.

Reply:
I just want to use this static parameter in wrapper class, and the DEFAULT_PAGE_SIZE name is more better, will change that.

Reply:
Oops, sorry, didn't notice that it's the static variable. `DEFAULT_PAGE_SIZE` is for global constant.

Reply:
np, I think so, didn't use the static variable in class for some time.
Change back, but default_page_size is still good than page_size.

Reply:
Don't use static member in the instance, this will lead to bugs if you use multiple `PDCClient` instances.